source/core/NumericString.cpp

```diff
@@ -224,11 +224,14 @@ void DefaultFormatCode(Int32 count, StaticTypeAndData arguments[], TempStackCStr
         }
         switch (argType->BitEncoding()) {
 
-        case kEncoding_UInt:
-        case kEncoding_Enum: {
+        case kEncoding_UInt: {
             buffer->AppendCStr("%u");
         }
         break;
+            case kEncoding_Enum: {
```

Comment:
Remove an extra tab, so the new case aligns with others. We need to consider adding linting sooner rather than later so this is found automatically for us.